epytope/test/TestEpitopeAssembly.py

```diff
@@ -20,7 +20,7 @@ def test_simple_assembly(self):
         :return:
         """
         pred = CleavageSitePredictorFactory("PCM")
-        assembler = EpitopeAssembly(self.peptides, pred, solver="cbc", verbosity=0)
+        assembler = EpitopeAssembly(self.peptides, pred, solver="glpk", verbosity=0)
```

Comment:
We changed this to `glpk` because of the installation issue with `cbc` in the tests right?

Reply:
Yes

Reply:
@b-schubert @lkuchenb Any ideas why we might have issues with the `cbc` installation? I assume it worked before.
Is it fine to switch to `glpk`?

Reply:
Noted in backlog